### PR TITLE
Начинаем методичную разметку личного проекта

### PR DESCRIPTION
**Pull request by Gozh78**
Сделана начальная разметка главной страницы, страницы каталога и страницы с формой.

---
:mortar_board: [Начинаем методичную разметку личного проекта](https://up.htmlacademy.ru/adaptive/18/user/1173773/tasks/2)

:boom: https://htmlacademy-adaptive.github.io/1173773-cat-energy-18/3/

### CODE REVIEW

File: source/index.html [hunk 1]
```diff
@@ -6,7 +6,110 @@
...
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
 
+    <main class="page-main">
+      <section class="nutrition-program">
+        <header class="nutrition-program__header">
+          <h1 class="nutrition-program__heading">Функциональное питание для котов</h1> <!--Возможно, лучше вынести из этой шапки-->
```
Comment (Dima-Dim) [inline source/index.html]: Вообще сам по себе header здесь странен :)

File: source/index.html [hunk 2]
```diff
@@ -6,7 +6,110 @@
...
+          <b class="nutrition-program__slogan">Занялся собой? Займись котом!</b>
+          <img class="nutrition-program__logo-image" src="https://via.placeholder.com/150x150" width="150" height="150" alt="Банка Cat Energy">
+          <a class="nutrition-program__find-programs button" href="form.html">Подобрать программу</a>
+        </header>
+        <div class="nutrition-program__wrapper">
+          <ul class="nutrition-program__list">
+            <li class="nutrition-program__item">
+              <h3 class="nutrition-program__heading">Похудение</h3>
+              <div class="nutrition-program__logo"></div> <!--Залью цветом, чтобы точно всегда этот круг появлялся при загрузке страницы, и добавлю иконку псевдоэлементом-->
```
Comment (Dima-Dim) [inline source/index.html]: Круг тоже может быть псевдоэлементом ;)

File: source/index.html [hunk 3]
```diff
@@ -6,7 +6,110 @@
...
+              <p class="nutrition-program__description">Ваш кот весит больше собаки и почти утратил способность лазить по деревьям?
+                Пора на диету! Cat Energy Slim поможет вашему питомцу сбросить лишний вес.</p>
+              <a class="nutrition-program__show-programs" href="catalog.html">Каталог slim</a><!-- Пока ссылка на каталог... Нужно создать еще две страницы catalog-slim(pro).html или будет реализация автоматического фильтра по ссылке? -->
```
Comment (Dima-Dim) [inline source/index.html]: https://up.htmlacademy.ru/adaptive/18/project/cat-energy#cat-energy-4-3
```
Страницы разделов реализовывать не нужно.
```

File: source/index.html [hunk 4]
```diff
@@ -6,7 +6,110 @@
...
+            </li>
+          </ul>
+        </div>
+      </section>
+
+      <section class="how-it-works">
+        <h2 class="how-it-works__heading">Как это работает</h2>
+        <ul class="how-it-works__list">
+          <li class="how-it-works__item">
+            <img class="how-it-works__logo-image" src="https://via.placeholder.com/150x150" width="150" height="150" alt="Польза Cat Energy">
```
Comment (Dima-Dim) [inline source/index.html]: Это не контентные изображения и даже не изображения, а иконки и поэтому они могут быть псевдоэлементами.

File: source/catalog.html [hunk 5]
```diff
@@ -0,0 +1,328 @@
...
+    </header>
+
+    <main class="page-catalog">
+      <h1 class="page-catalog__heading">Каталог продукции</h1>
+      <section class="catalog">
+        <ul class="catalog__list">
+          <li class="catalog__item">
+            <h3>Cat Energy PRO 500 г</h3>
+            <img class="catalog__item-image" src="https://via.placeholder.com/150x150" width="150" height="150" alt="Cat Energy PRO 500 г">
+            <table class="catalog__item-description"> <!--Можно ли так сделать? Это не модификатор, а просто более уточняющее название-->
```
Comment (Dima-Dim) [inline source/catalog.html]: Да, можно.
Что касается таблицы, если её и делать, то с «шапкой» в которой будут указаны названия столбцов. Если смотреть «ширие», то весь этот список по сути таблица (прайс) товаров. Но делать это таблицей я тебя не заставляю.

File: source/catalog.html [hunk 6]
```diff
@@ -0,0 +1,328 @@
...
+      </nav>
+    </header>
+
+    <main class="page-catalog">
+      <h1 class="page-catalog__heading">Каталог продукции</h1>
+      <section class="catalog">
+        <ul class="catalog__list">
+          <li class="catalog__item">
+            <h3>Cat Energy PRO 500 г</h3>
+            <img class="catalog__item-image" src="https://via.placeholder.com/150x150" width="150" height="150" alt="Cat Energy PRO 500 г">
```
Comment (Dima-Dim) [inline source/catalog.html]: https://up.htmlacademy.ru/adaptive/18/project/cat-energy#cat-energy-6-1
```
Изображение и название товара – ссылки на страницу с описанием товара. Страницу с описанием товара реализовывать не нужно.
```

File: source/catalog.html [hunk 7]
```diff
@@ -0,0 +1,328 @@
...
+                  500 Р.
+                </td>
+              </tr>
+            </table>
+            <a class="catalog__make-order button" href="#">Заказать</a>
+          </li>
+          <li class="catalog__item catalog__item--disappearing">
+            <b class="catalog__more-items">Показать еще 100500 товаров</b>
+            <p class="catalog__more-tastes">На самом деле вкусов гораздо больше!</p>
+            <button class="catalog__show-all button" type="button">Показать все</button>
```
Comment (Dima-Dim) [inline source/catalog.html]: https://up.htmlacademy.ru/adaptive/18/project/cat-energy#cat-energy-6-3
```
Кнопка «Показать все» показывает новые товары, загружаемые по ajax. Загрузку по ajax реализовывать не нужно. При отключенном JS должен осуществляться переход на страницу новых товаров (страницу новых товаров реализовывать не нужно).
```

File: source/catalog.html [hunk 8]
```diff
@@ -0,0 +1,328 @@
...
+          </li>
+        </ul>
+      </section>
+    </main>
+    <!--Есть ещё вариант для элементов каталога:
+
+    <ul class="catalog__list">
+      <li class="catalog__item catalog-item">   Здесь начинаются изменения - добавил класс
```
Comment (Dima-Dim) [inline source/catalog.html]: Этот вариант более универсальный и может быть более независимым по части карточек товаров, если проект большой и нечто подобное может встречаться где-то ещё. Но в данном случае не принципиально, можно сделать как удобнее. Ну и нужно иметь ввиду что https://up.htmlacademy.ru/adaptive/18/criteries#b13
```
нужно разделить код таким образом, чтобы каждый блок был в своём файле
```

File: source/index.html (hunk 1 above)
Comment (Gozh78) [reply source/index.html]: В лекции просто видел, как в демке делается хэдер примерно так же)

Comment (Gozh78) [reply source/index.html]: Тогда ни во что не оборачивать код?

File: source/index.html (hunk 2 above)
Comment (Gozh78) [reply source/index.html]: Я просто подумал, что на случай, если по какой-то причине не прогрузится псевдоэлемент, то всегда будет всё равно виден хотя бы круг рядом с заголовком, чтобы гармонично сам заголовок смотрелся на фоне текста (это особенно заметно с блоком "Набор массы", если убрать мысленно круг).
Или же раз не прогружаются псевдоэлементы, то не прогрузятся и все остальные стили, и тогда нет толку от того, что я сделал круг дивом?:)

File: source/index.html (hunk 1 above)
Comment (Dima-Dim) [reply source/index.html]: Пока обёртка не понадобилась можно не оборачивать.

File: source/index.html (hunk 2 above)
Comment (Dima-Dim) [reply source/index.html]: В этом случае сомнительная выгода, которой скорее всего не будет. Ведь стили для div-а в любом случае в css, т.е. там же где и псевдоэлемент.

File: source/index.html (hunk 3 above)
Comment (Gozh78) [reply source/index.html]: Я имею в виду то, что под каждый раздел каталога своя страница же? То есть я страницы реализовывать не буду, но ведь файлы catalog-slim.html и catalog-pro.html создам, верно?

File: source/catalog.html (hunk 5 above)
Comment (Gozh78) [reply source/catalog.html]: То есть создать шапку, элементы которой просто будут доступно спрятаны?

Вы говорите про список карточек товаров? А разве не лучше будет сделать на флексах вместо таблицы? Ведь с таблицей будет что-то непонятное твориться, наверное, когда будет изменяться ширина сайта.

File: source/index.html (hunk 3 above)
Comment (Dima-Dim) [reply source/index.html]: В этом плане да, как-то так могло бы быть.

File: source/catalog.html (hunk 5 above)
Comment (Dima-Dim) [reply source/catalog.html]: Про "шапку" да.

Таблица тоже может быть флексами :) На примере Барбершопа это должны будут показывать, если не будет изменений в демках.
